Packs/ExpanseV2/README.md

```diff
@@ -1,9 +1,11 @@
-The Expanse content pack for Cortex XSOAR provides full coverage of the Expanse Expander and Behavior product capabilities to allow SOCs to automate the defense of their company's attack surface. The integrations included in the pack enable fetching and mirroring of Expanse Issues into Cortex XSOAR incidents, and ingestion of indicators (IPs, domains, and certificates) referring to the corporate network perimeter as discovered by Expanse, a Palo Alto Networks company.
+The Cortex Xpanse (previously **Expanse v2**) pack for Cortex XSOAR provides full coverage of the Expanse Expander and Behavior product capabilities to allow SOCs to automate the defense of their company's attack surface. The integrations included in the pack enable fetching and mirroring of Expanse Issues into Cortex XSOAR incidents, and ingestion of indicators (IPs, domains, and certificates) referring to the corporate network perimeter as discovered by Expanse, a Palo Alto Networks company.
 
 Through a powerful set of playbooks, analysts can correlate the discovered information with data provided from internal security systems (Palo Alto Networks Cortex Data Lake, Prisma Cloud, and Panorama, Active Directory, Splunk SIEM, etc.) to help pinpoint the right owners of assets and automate remediation.
 
+Pack and integration named **Expanse v2** were changed to **Cortex Xpanse**. All other content items are still named the same.
```

Comment:
let's move this line to the top of the page.
```suggestion
> <b>Note<\b>: This Pack, as well as its previously named **Expanse v2** Integration, were renamed to **Cortex Xpanse**. All other content items are still named the same.
```
see [Okta IAM](https://xsoar.pan.dev/docs/reference/integrations/okta-iam) note at the top of the page as an example for what is the `>` at the beginning of the line doing